docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -94,32 +91,36 @@ spec:
       secret: osp-secret
 '
 ----
+
+. Wait for the {bare_metal} control plane services CRs to become ready:
 +
-The operator begins to apply the configuration and start the necessary Bare Metal Provisioning services. Once the services have reached a running state, the {bare_metal} automatically begins polling the power state of bare metal nodes for which it is configured to manage.
+----
+$ oc wait --for condition=Ready --timeout=300s ironics.ironic.openstack.org ironic
+----
 
-. Wait for Bare Metal Provisioning control plane services' custom resources to become ready:
+. Optional: Verify that the individual services are ready:
```

Comment:
I split step 2 into two steps. Step 3 is optional. Please confirm that this is accurate.

Reply:
Well, technically it is optional since "ironic" should not be "Ready" unless the other services are.
But - **I think it is better to be verbose - do the extra verification of the individual services to make sure.**

Historically this has been an issue with upgrade documentation, it would instruct the use to run a command - but there would be nothing to confirm the results where successful. Customers would run said command, it would not do what is expected and in some cases not report an error, the lack of step to validate the result made the customer move on with the upgrade only to fail at a later step and in many cases end up with support escalations.

Reply:
Ok that makes sense. I'll remove "Optional" so it is more explicit: "Verify that the individual services are ready:"